snapshot/lib/assets/SnapfileTemplate

```diff
@@ -23,8 +23,15 @@
 # Where should the resulting screenshots be stored?
 # output_directory "./screenshots"
 
-# remove the '#' to clear all previously generated screenshots before creating new ones
-# clear_previous_screenshots true
+
+# Where should the resulting videos be stored?
```

Comment:
this comment should take into account that by default no video will be create. So probably somethign like